docs/src/manual.md

```diff
@@ -2,6 +2,8 @@
 
 ## Links
 
+Links are reparametrizations utilized in the estimation. For example, if we want to estimate a parameter ``\theta`` which is by definition strictly positive, then an obvious way to estimate ``\theta`` via numerical optimization is to model ``\tilde{\theta} = \ln{\theta}``. We refer to this procedure as **linking**. After obtaining the optimal value of ``\tilde{\theta}``, we can then **unlink** it to obtain ``\theta``.
```

Comment:
I prefer to define this as `f` instead of `\theta`because in the paper I assumed `f` as time-varying parameter and theta as elements of `A`, `B` and `\omega`